src/tools/create-require.js

```diff
@@ -0,0 +1,21 @@
+'use-strict';
+
+const { dependencies, devDependencies } = require('../../package.json');
+const { RequireModuleError } = require('../errors');
+
+const createRequire = () => module => {
+  const libs = Object.keys(dependencies).concat(Object.keys(devDependencies));
+  const allowedLibs = ['lodash'];
```

Comment:
since godzilla apps can't depend on things directly, `lodash` can only be required as a happy side effect of `core` already depending on it. not sure there's explicit change needed here, but it was worth mentioning.

Reply:
Yep, this was all just an idea to toss out. I figured throwing out a restricted version would be easier to show than try explaining.

In going through all this I've been getting a better handle on how apps are loaded and executed in `core`.

Reply:
totally! i've learned a lot too, I really appreciate you going through this. It's some pretty wild stuff!

Reply:
@stevelikesmusic so I think i've flopped here - it's probably worth blocking anything in the `core` requirements. _maybe_ an exception for lodash since we're unlikely to not depend on it and it's super useful, but that might cause more confusion than help.

Reply:
yeah I think restricting to just core node is the way to go. If you know lodash, or find yourself reaching or it, you should probably be on the CLI.

Reply:
very fair. I'll grab this next week and ping you on the review. thanks!